packages/mui-material/src/styles/experimental_extendTheme.js

```diff
@@ -53,25 +69,106 @@ export default function extendTheme(options = {}, ...args) {
     const palette = theme.colorSchemes[key].palette;
 
     // attach black & white channels to common node
-    if (key === 'dark') {
-      palette.common.background = palette.common.background || '#000';
-      palette.common.onBackground = palette.common.onBackground || '#fff';
+    if (key === 'light') {
+      setColor(palette.common, 'background', '#fff');
+      setColor(palette.common, 'onBackground', '#000');
     } else {
-      palette.common.background = palette.common.background || '#fff';
-      palette.common.onBackground = palette.common.onBackground || '#000';
+      setColor(palette.common, 'background', '#000');
+      setColor(palette.common, 'onBackground', '#fff');
+    }
+
+    // assign component variables
+    assignNode(palette, [
+      'AppBar',
+      'Chip',
+      'FilledInput',
+      'LinearProgress',
+      'Skeleton',
+      'Slider',
+      'SnackbarContent',
+      'StepConnector',
+      'StepContent',
+      'Switch',
+      'TableCell',
+      'Tooltip',
+    ]);
+    if (key === 'light') {
+      setColor(palette.AppBar, 'defaultBg', 'var(--mui-palette-grey-100)');
+      setColor(palette.Chip, 'defaultBorder', 'var(--mui-palette-grey-400)');
+      setColor(palette.Chip, 'defaultAvatarColor', 'var(--mui-palette-grey-700)');
+      setColor(palette.Chip, 'defaultIconColor', 'var(--mui-palette-grey-700)');
+      setColor(palette.FilledInput, 'bg', 'rgba(0, 0, 0, 0.06)');
+      setColor(palette.FilledInput, 'hoverBg', 'rgba(0, 0, 0, 0.09)');
+      setColor(palette.FilledInput, 'disabledBg', 'rgba(0, 0, 0, 0.12)');
+      setColor(palette.LinearProgress, 'primaryBg', lighten(palette.primary.main, 0.62));
+      setColor(palette.LinearProgress, 'secondaryBg', lighten(palette.secondary.main, 0.62));
+      setColor(palette.LinearProgress, 'errorBg', lighten(palette.error.main, 0.62));
+      setColor(palette.LinearProgress, 'infoBg', lighten(palette.info.main, 0.62));
+      setColor(palette.LinearProgress, 'successBg', lighten(palette.success.main, 0.62));
+      setColor(palette.LinearProgress, 'warningBg', lighten(palette.warning.main, 0.62));
+      setColor(palette.Skeleton, 'bg', 'rgba(var(--mui-palette-text-primary) / 0.11)');
```

Comment:
```suggestion
      setColor(palette.Skeleton, 'bg', 'rgba(var(--mui-palette-text-primaryChanel) / 0.11)');
```

It doesn't work right now - https://deploy-preview-32835--material-ui.netlify.app/experiments/material-ui/skeleton/